pyaedt/hfss3dlayout.py

```diff
@@ -1820,6 +1824,9 @@ def edit_hfss_extents(
         if not air_truncate_model_at_ground_layer == "keep":
             arg.append("TruncAtGnd:=")
             arg.append(air_truncate_model_at_ground_layer)
+        if air_horizontal_padding:
+            arg.append("AirHorExt:=")
+            arg.append(["Ext:=", str(air_vertical_positive_padding), "Dim:=", airbox_values_as_dim])
```

Comment:
Where is `air_horizontal_padding` value used?

Reply:
@MaxJPRey good catch. I fixed it